src/vea/vea_util.c

```diff
@@ -356,6 +356,20 @@ vea_metrics_count(void)
 	return (sizeof(struct vea_metrics) / sizeof(struct d_tm_node_t *));
 }
 
+void
+vea_metrics_nvme_used_set(void* metrics, struct d_tm_node_t* nvme_used)
```

Comment:
(style)  "foo* bar" should be "foo *bar"
(style)  "foo* bar" should be "foo *bar"